### ISSUE TITLE
Error trying to run the react-load demo

### ISSUE THREAD

**Issue by dariov1988** [open-telemetry/opentelemetry-js-contrib#654]
### What version of OpenTelemetry are you using?
0.25.0

### What version of Node are you using?
v14.16.0

### What did you do?
I made a clean clone of the project
go to: examples/react-load/react and follow the Readme instructions:
run: npm install
run: npm run docker:start (zipkin start ok)
run: npm run build (complains about opentelemetry/api but after install it the build finish successfully)
run: npm install serve -g
run npm start

### What did you expect to see?
The demo app running.

### What did you see instead?
The app do not load and show an error in the console:
`Uncaught TypeError: Cannot read property 'debug' of undefined`

### Additional context
<img width="1919" alt="error" src="https://user-images.githubusercontent.com/10393939/131904952-ff1cab1d-5bb9-46ae-919f-e00827461a6a.png">

**Comment by obecny** [comment]
just checked, the example is broken, but even though some strange thing is happening with react, can reproduce this bug

**Comment by jussikuosa** [comment]
I added this to my `web-tracer.js` setup as a workaround: 

```
import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api';
// ...
const diagLogger = new DiagConsoleLogger();
diag.setLogger(diagLogger, DiagLogLevel.ALL);
// ...
BaseOpenTelemetryComponent.setTracer(serviceName);
BaseOpenTelemetryComponent.setLogger(diagLogger); // provider.logger);
```

**Comment by nemoshlag** [comment]
Hi I want to contribute to otl, and this seems like a good first issue, you can assign this for me thanks

**Comment by dyladan** [comment]
Fixed in #764
